### ISSUE TITLE
Publish pulumi-command artifact to mvn central

### ISSUE THREAD

**Issue by rururu387** [pulumi/pulumi-command#612]
Hi!
TL;DR: Couldn't find mvn artifact for pulumi-command on the web. Thank you for your project!

I'm trying to use pulumi (java) for a pet-project. Wanted my first pulumi app to simply execute shell program to print hello-world but couldn't find mvn dependency that contains all the classes described in [documentation](https://www.pulumi.com/registry/packages/command/#a-simple-local-resource-random). It's neither in the [mvnrepository](https://mvnrepository.com/artifact/com.pulumi/pulumi-command) nor in the [sonatype repository](https://s01.oss.sonatype.org/#nexus-search;quick~pulumi-command). It would be great if you publish it. Or please specify where to search for it in documentation. In case I've blundered it somehow - sorry for bothering.

Thanks!

**Comment by rururu387** [comment]
https://mvnrepository.com/artifact/com.pulumi/command is an actual url =)
